src/tito/cli.py

```diff
@@ -697,7 +698,7 @@ def main(self, argv):
         propsfile = os.path.join(rel_eng_dir, TITO_PROPS)
         if not os.path.exists(propsfile):
             if not os.path.exists(rel_eng_dir):
-                getoutput("mkdir -p %s" % rel_eng_dir)
+                mkpath(rel_eng_dir)
```

Comment:
Why not use [`os.makedirs()`](https://docs.python.org/3/library/os.html#os.makedirs)?

Reply:
True, what was I thinking. Fixed.
Thank you @Conan-Kudo